runtime-modules/content/src/lib.rs

```diff
@@ -1232,18 +1233,18 @@ decl_module! {
             origin,
             actor: ContentActor<T::CuratorGroupId, T::CuratorId, T::MemberId>,
             proof: Vec<ProofElement<T>>,
-            item: PullPayment<T>,
-            destination: T::AccountId,
+            item: PullPayment<T>
         ) -> DispatchResult {
             let (channel, reward_account, amount) =
                 Self::ensure_can_claim_channel_reward(&origin, &actor, &item, &proof)?;
 
-            // Ensure features are not paused
+            // Ensure withdrawals are not paused
             channel.ensure_feature_not_paused::<T>(PausableChannelFeature::ChannelFundsTransfer)?;
-            channel.ensure_feature_not_paused::<T>(PausableChannelFeature::CreatorCashout)?;
```

Comment:
What was the issue with claims being blocked by the paused feature check, exactly ?

Reply:
There are 2 extrinsics that allow claiming channel reward: `claim_channel_reward` and `claim_and_withdraw_channel_reward`. Since `PausableChannelFeature::CreatorCashout` should block both I moved this `ensure` check from `claim_and_withdraw_channel_reward` to `ensure_can_claim_channel_reward`